src/hearings/models/hearingResponse.model.ts

```diff
@@ -8,6 +8,6 @@ export interface HearingResponseModel {
   laCaseStatus: LaCaseStatus;
   listingStatus: ListingStatus;
   hearingCancellationReason: string;
-  hearingDaySchedule: HearingDayScheduleModel;
+  hearingDaySchedule: HearingDayScheduleModel[];
```

Comment:
if we can also amend the /api/hearings/models/hearingResponse.model.ts.model.ts to keep them consistent.